cpp/include/cudf/lists/detail/dremel.hpp

```diff
@@ -195,4 +195,20 @@ dremel_data get_dremel_data(column_view h_col,
                             bool output_as_byte_array,
                             rmm::cuda_stream_view stream);
 
+/**
+ * @brief Get Dremel offsets, repetition levels, and modified definition levels to be used for
+ *        lexicographical comparators. The modified definition levels are produced by treating
+ *        each nested column in the input as nullable
+ *
+ * @param h_col Column of LIST type
+ * @param nullability Pre-determined nullability at each list level. Empty means infer from
+ * `col`
+ * @param output_as_byte_array flag to indicate if output should be a byte array
```

Comment:
"if output should be a byte array" but what otherwise?

Reply:
Good question. I will have to review the code for what this flag does, unfortunately the current version does not document this flag either